src/VirtoCommerce.CatalogModule.Core/Model/ExcludedProperty.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.CatalogModule.Core.Model
+{
+    public class ExcludedProperty
```

Comment:
Better inherit this type from `ValueObject`   then you not need to override any Equals and GetHashCode methods.